### PR TITLE
fixed #139

### LINKED ISSUE #139: Change links and text to the current repository name
### Description
As a contributor,
I need all the links and text that is using` open-source-programs-backend` changed into `anitab-forms-backend`,
so that there is no confusion and 404 pages for newcomers.

### Update Required

- [ ] Change open-source-programs-backend to anitab-forms-backend.

### Definition of Done
 

- [ ] All of the required items are completed.

  

- [ ]   Approval by 1 mentor.

### Estimation

3 hours.

### PR DESCRIPTION
**Pull request by untari**
### Description
I have changed the text and link in the README.md file and docs file according to the current repository name from  open-source-programs-backend to anitab-forms-backend .

Fixes #139

### Type of Change:
- Documentation

### CODE REVIEW

**Comment by codesankalp** [comment]
@untari Please update the PR description and title to something relevant and meaningful.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Open Source Programs (Backend)
+# Anitab Forms Backends
```
Comment (codesankalp) [inline README.md]:
```suggestion
# Anitab Forms Backend
```

**Review by codesankalp — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -225,7 +225,7 @@ This project follows the [all-contributors](https://github.com/all-contributors/
 Contributions of any kind welcome!
 
 ## Contact
-You can reach the admins, maintainers and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). If you are interested in contributing to the OSP project, you may join the stream [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) and ask questions or intereact with the community. Join Us!
+You can reach the admins, maintainers and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). If you are interested in contributing to the OSP project, you may join the stream [#anitab-forms-backend](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) and ask questions or intereact with the community. Join Us!
```
Comment (isabelcosta) [inline README.md]:
```suggestion
You can reach the admins, maintainers and our community on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). If you are interested in contributing to the OSP project, you may join the stream [#anitab-forms](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) and ask questions or intereact with the community. Join Us!
```

**Review by isabelcosta — APPROVED** [review]
Thank you for contributing @untari 💯

**Review by codesankalp — APPROVED** [review]

**Pull request merged**